cdk/lib/payment-api.test.ts

```diff
@@ -0,0 +1,16 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert";
+import { App } from "@aws-cdk/core";
+import { PaymentApi } from "./payment-api";
+
+describe("The Frontend stack", () => {
```

Comment:
Is 'Frontend' right here?

Reply:
Good spot! Should be `The Payment API stack`